devtools-ci-index.yaml

```diff
@@ -556,6 +556,10 @@
                 - openshift-connector
     triggers:
         - github
+        - github-pull-request:
```

Comment:
Then, how do you want to try/test that it is working. If I would create at first a PR for verification job, the job would not work because it is depending on the code which you want to merge as a second PR. It is tightly coupled.

Also testing that it works is not possible without merging it whole.

I can add white list of persons or make an org white-listed - what is better for you?

Trigger phase could be anything we agree on. On other project we were using "test this" "test this please" "testPr" etc... But if you are afraid that someone white-listed could accidentally trigger the job by writing a comment "test PR", I can change it.